tuna/miopen/scripts/report.py

```diff
@@ -0,0 +1,200 @@
+#!/usr/bin/env python3
+###############################################################################
+#
+# MIT License
+#
+# Copyright (c) 2023 Advanced Micro Devices, Inc.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+#
+###############################################################################
+"""script for detecting find db entries with missing perf db entries"""
+
+import pandas as pd
+from tuna.parse_args import TunaArgs, setup_arg_parser
+from tuna.utils.logger import setup_logger
+from tuna.miopen.db.tables import MIOpenDBTables
+from tuna.dbBase.sql_alchemy import DbSession
+from tuna.utils.db_utility import get_id_solvers
+
+LOGGER = setup_logger('report')
+
+
+def parse_args():
+  """command line parsing"""
+  parser = setup_arg_parser(
+      'Post tuning report on config performance for current session',
+      [TunaArgs.CONFIG_TYPE])
+  parser.add_argument(
+      '--session_id',
+      required=True,
+      dest='session_id',
+      type=int,
+      help='Session id to be used for comaprison against golden_v')
+  parser.add_argument('--golden_v',
+                      dest='golden_v',
+                      required=True,
+                      type=int,
+                      default=None,
+                      help='Target golden miopen version')
+
+  args = parser.parse_args()
+  return args
+
+
+def get_data(args, dbt):
+  """Get data from DB based on args.session_id and optional golden_v"""
+
+  with DbSession() as session:
+    query = f"select t1.config as c1, t1.solver as s1, t1.kernel_time as k1,"\
```

Comment:
@JehandadKhan  query to get the golden_v and session_v matched by session_arch and num_cu